coop_point_of_sale/models/pos_session.py

```diff
@@ -29,25 +29,48 @@ class PosSession(models.Model):
         string='Day (Search)', compute='_compute_date_search',
         multi='_date_search', store=True, index=True)
 
-    order_qty = fields.Integer(
-        compute='_compute_orders', string='Orders Qty', store=True)
+    order_count = fields.Integer(
+        compute='_compute_orders',
+        string='Orders Count',
+        store=True
+    )
 
-    amount_subtotal = fields.Monetary(
-        compute='_compute_orders', string='Amount Subtotal', store=True)
+    amount_untaxed = fields.Monetary(
+        compute='_compute_orders',
+        string='Amount Untaxed',
+        store=True
+    )
 
-    total_amount = fields.Monetary(
-        compute='_compute_orders', string='Transactions Total', store=True)
+    amount_total = fields.Monetary(
+        compute='_compute_orders',
+        string='Amount Total',
+        store=True
+    )
 
     @api.multi
-    @api.depends('order_ids.lines.price_subtotal_incl',
-                 'order_ids.lines.price_subtotal')
+    @api.depends('order_ids.amount_total', 'order_ids.amount_tax')
     def _compute_orders(self):
+        session_data = self.env['pos.order'].read_group([
+            ('state', '!=', 'cancel'),
+            ('session_id', 'in', self.ids)],
+            ['amount_total', 'amount_tax'],
+            ['session_id']
```

Comment:
```suggestion
            fields=['amount_total', 'amount_tax'],
            groupby=['session_id'],
```

Much better if you use kwargs here so it's easier to read, otherwise it gets mixed up with the domain